### ISSUE TITLE
Array of type Enum not created in generated Schema

### ISSUE THREAD

**Issue by jihdeh** [prisma/upgrade#47]
I have a prisma1 schema that looks like this.

```
type User {
  id: UUID! @id @unique
  email: String! @unique
  name: String!
  displayName: String!
  password: String!
  profile: UserProfile @relation(name: "UserProfile", link: INLINE, onDelete: CASCADE)
}

type UserProfile {
  id: UUID! @id @unique
  user: User! @relation(name: "UserProfile", onDelete: CASCADE)
  interests: [InterestOptions!] @scalarList(strategy: RELATION)

  createdAt: DateTime! @createdAt
  updatedAt: DateTime! @updatedAt
}

enum InterestOptions {
  FASHION
  TRAVEL_AND_HOSPITALITY
  COFFEE
}
```

When I run the commands to update the schema, the generated schema looks like this.

```
model User {
  id                     String   @default(uuid()) @id
  email               String   @unique
  name               String
  displayName  String
  password        String
  profile             String?
  Profile             UserProfile?   @relation(fields: [profile], references: [id])
}

model UserProfile {
  id                                                        String                             @default(uuid()) @id
  User                                                   User
  UserProfile_interests                        UserProfile_interests[]
}

*Enum type of InterestOptions was not created*
```

The error I got in running an SQL query
```
=> ALTER TABLE "default$default"."UserProfile" ADD UNIQUE ("user");

*ERROR:  column "user" named in key does not exist*
```
Version
"prisma-upgrade": "^0.0.26",
"@prisma/cli": "^2.2.0"

While that happened, can i just manually go ahead and update the model like this?:

```
model UserProfile {
  id                             String                             @default(uuid()) @id
  User                        User
  interests                 interestsOptions[]
}

enum InterestOptions {
  FASHION
  TRAVEL_AND_HOSPITALITY
  COFFEE
}
```

**Comment by matthewmueller** [comment]
Thanks for reporting! I'll investigate this issue and get back to you soon.

**Comment by matthewmueller** [comment]
Are you using MySQL or Postgres?

**Comment by jihdeh** [comment]
Postgres

**Comment by matthewmueller** [comment]
Okay this should be fixed by #57. Please give it a try in 0.0.33.

**Comment by c316** [comment]
@matthewmueller I'm having the same issue. 


```
  Fix 1-1 relations by adding `UNIQUE` constraints
  https://pris.ly/d/schema-incompatibilities#inline-1-1-relations-are-recognized-as-1-n-missing-unique-constraint

    ALTER TABLE "default$default"."Location" ADD UNIQUE ("activity");
    ALTER TABLE "default$default"."ContactSettings" ADD UNIQUE ("contact");
```

My Prisma 1 datamodel file has this relevant section

```

type Activity {
  id: ID! @id
  location: Location @relation(name: "ActivityLocation", onDelete: CASCADE, link: INLINE)
}
type Location {
  activity: Activity @relation(name: "ActivityLocation", onDelete: SET_NULL)
  countryCode: String
  latitude: Float
  longitude: Float
  name: String
  id: ID! @unique @id
}


type Contact {
  address: String
  id: ID! @id
  settings: ContactSettings @relation(name: "ContactSettings", onDelete: CASCADE, link: INLINE)
}
type ContactSettings {
  doNotContact: Boolean
  contact: Contact! @relation(name: "ContactSettings", onDelete: SET_NULL)
  id: ID! @id
}
```
